core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -420,27 +424,28 @@ object ConsumerGroupCommand {
                                   .ofType(classOf[String])
     val options = parser.parse(args : _*)
 
+    val useOldConsumer = options.has(zkConnectOpt)
+
     val allConsumerGroupLevelOpts: Set[OptionSpec[_]] = Set(listOpt, describeOpt, deleteOpt)
 
     def checkArgs() {
       // check required args
-      if (options.has(newConsumerOpt)) {
+      if (useOldConsumer) {
+        CommandLineUtils.checkRequiredArgs(parser, options, zkConnectOpt)
+
+        if (options.has(bootstrapServerOpt))
+          CommandLineUtils.printUsageAndDie(parser, s"Option $bootstrapServerOpt is not valid with old consumer.")
```

Comment:
This message may be confusing because users have not specified `old consumer`, they just specified `zookeeper`. There are a few other cases like this.